server/service/src/sync/sync_status/test.rs

```diff
@@ -1,13 +1,16 @@
 use std::{collections::HashMap, sync::Arc};
```

Comment:
Changes in this file are pretty much all format, i had a very long string previously and cargo fmt fails to format file if there is a long string inside chaining : /.